src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -54,59 +55,29 @@
         QUERY_DATE, "2020-08-15", "2020-08-16", "2020-08-17", "2020-08-18"
     );
 
-    /**
-     * Timestamps (in OTP's timezone) to test whether an itinerary is same-day as QUERY_DATE. These are all set after
-     * the OTP_TIMEZONE config value is known after the middleware is setup.
-     */
-    public static long _2020_08_12__03_00_00; // Aug 12, 2020 3:00:00 AM
-    public static long _2020_08_12__23_59_59; // Aug 12, 2020 11:59:59 PM
-    public static long _2020_08_13__02_59_59; // Aug 13, 2020 2:59:59 AM, considered to be Aug 12.
-    public static long _2020_08_13__03_00_00; // Aug 13, 2020 3:00:00 AM
-    public static long _2020_08_13__23_59_59; // Aug 13, 2020 11:59:59 PM
-    public static long _2020_08_14__02_59_59; // Aug 14, 2020 2:59:59 AM, considered to be Aug 13.
-    public static long _2020_08_14__03_00_00; // Aug 14, 2020 3:00:00 AM
+    /** Timestamps (in OTP's timezone) to test whether an itinerary is same-day as QUERY_DATE. */
+    public static final long _2020_08_12__03_00_00 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 12, 3, 0, 0)); // Aug 12, 2020 3:00:00 AM
+    public static final long _2020_08_12__23_59_59 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020,8,12,23,59,59)); // Aug 12, 2020 11:59:59 PM
+    public static final long _2020_08_13__02_59_59 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 13, 2, 59, 59)); // Aug 13, 2020 2:59:59 AM, considered to be Aug 12.
+    public static final long _2020_08_13__03_00_00 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 13, 3, 0, 0)); // Aug 13, 2020 3:00:00 AM
+    public static final long _2020_08_13__23_59_59 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 13, 23, 59, 59)); // Aug 13, 2020 11:59:59 PM
+    public static final long _2020_08_14__02_59_59 = otpDateTimeAsEpochMillis(LocalDateTime.of(
+        2020, 8, 14, 2, 59, 59)); // Aug 14, 2020 2:59:59 AM, considered to be Aug 13.
+    public static final long _2020_08_14__03_00_00 = otpDateTimeAsEpochMillis(LocalDateTime.of(
```

Comment:
I think we could improve readability by just changing the arg of `otpDateTimeAsEpochMillis` to match those of `LocalDateTime#of` (i.e., year, month, etc.), so that we're not repeating `LocalDateTime.of(` over and over. But I don't feel too strongly about this.

Reply:
Refactored in 836dc4901ea2d8e2ba609c220a48c6ab196ed326